### PR TITLE
[Docs] ifp documentation

### PR DESCRIPTION
**Pull request by luarss**
Fix missing documentation and formatting issues

https://openroad--3716.org.readthedocs.build/en/3716/main/src/ifp/README.html

### CODE REVIEW

File: src/ifp/README.md [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
-  -utilization util               utilization (0-100 percent)
-  [-aspect_ratio ratio]           height / width, default 1.0
-  [-core_space space
-    or "bottom top left right"]   space around core, default 0.0 (microns).
-                                  Should be either one value for all margins
-                                  or 4 values, one for each margin.
+  [-utilization util]
+  [-aspect_ratio ratio]
+  [-core_space space | {bottom top left right}]
+  [-die_area {lx ly ux uy}]
```
Comment (vijayank88) [inline src/ifp/README.md:19]: @luarss 
`-die_area`/`-core_area` used for fixed value from user.
`-utilization`/`-aspect_ratio` used for automatic die size calculation based on synthesis results. 
So split both with `or` word.

File: src/ifp/README.md [hunk 2]
```diff
@@ -35,45 +51,71 @@ die =  ( 0, 0 )
...
 
-With no arguments `make_tracks` adds X and Y tracks for each routing layer.
-With a `-layer` argument `make_tracks` adds X and Y tracks for layer with
-options to override the LEF technology X and Y pitch and offset.
+#### Options
 
-### Place pins around core boundary
+| Switch Name | Description |
+| ----- | ----- |
+| `layer` | select layer name to make tracks for (default all layers). |
```
Comment (vijayank88) [inline src/ifp/README.md:71]:
```suggestion
| `-layer` | select layer name to make tracks for (default all layers). |
```

**Review by vijayank88 — CHANGES_REQUESTED** [review]

File: src/ifp/README.md (hunk 1 above)
Comment (luarss) [reply src/ifp/README.md:19]: Do you recommend we change the function signature or just indicate in docs?

File: src/ifp/README.md (hunk 2 above)
Comment (luarss) [reply src/ifp/README.md:71]: Layer is not a keyword arg so I didn't add a `-` in front of it

File: src/ifp/README.md (hunk 1 above)
Comment (vijayank88) [reply src/ifp/README.md:19]: I don't understand your question. But you may refer following example and update it to use either one at a time for `initialize_floorplan`
with die/core area: https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ifp/test/init_floorplan1.tcl
with utilization: https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ifp/test/init_floorplan2.tcl#L7

File: src/ifp/README.md (hunk 2 above)
Comment (vijayank88) [reply src/ifp/README.md:71]: ok, got it. you can resolve this.

File: src/ifp/README.md (hunk 1 above)
Comment (luarss) [reply src/ifp/README.md:19]: I mean like do we want to force a either-or argument like this 
initialize_floorplan
    [-die_area die_area | -core_area core_area]

Comment (vijayank88) [reply src/ifp/README.md:19]: Maybe following format:
```
initialize_floorplan
  [-utilization util]
  [-aspect_ratio ratio]
  [-core_space space | {bottom top left right}]
   [-sites site_name]
or
initialize_floorplan
  [-die_area {lx ly ux uy}]
  [-core_area {lx ly ux uy}]
  [-sites site_name]
```

Comment (luarss) [reply src/ifp/README.md:27]: @vijayank88 Updated

File: src/ifp/README.md [hunk 3]
```diff
@@ -1,27 +1,51 @@
...
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-utilization` | Percentage utilization (0-100). |
+| `-aspect_ratio` | Ratio $\frac{height}{width}$ (default 1.0, float). |
+| `-core_space` | Space around core, default 0.0 microns. should be either one value for all margins or 4 values, one for each margin. |
+| `-die_area` | Die area coordinates in microns (lower left and upper right x-/y- coordinates). |
+| `-core_area` | Core area coordinates in microns (lower left and upper right x-/y- coordinates). |
+| `-sites` | Tcl list of sites to make rows for (e.g. `{SITEXX, SITEYY}`) |
```
Comment (vvbandeira) [inline src/ifp/README.md:41]: Only two allowed or should we add `, ...`?

**Review by vvbandeira — CHANGES_REQUESTED** [review]

File: src/ifp/README.md [hunk 4]
```diff
@@ -1,27 +1,51 @@
...
 
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-utilization` | Percentage utilization (0-100). |
+| `-aspect_ratio` | Ratio $\frac{height}{width}$ (default 1.0, float). |
+| `-core_space` | Space around core, default 0.0 microns. should be either one value for all margins or 4 values, one for each margin. |
+| `-die_area` | Die area coordinates in microns (lower left and upper right x-/y- coordinates). |
+| `-core_area` | Core area coordinates in microns (lower left and upper right x-/y- coordinates). |
```
[the remark this answers is not in the document]
Comment (luarss) [reply src/ifp/README.md:40]: Is this alright? 

```tcl
initialize_floorplan
  [-die_area {llx lly urx ury}]
  [-core_area {llx lly urx ury}]
  [-sites site_name]
```

File: src/ifp/README.md [hunk 5]
```diff
@@ -1,27 +1,62 @@
...
+- Parameters without square brackets `-param2 param2` are required.
+```
+
 ### Initialize Floorplan
 
+Do note that there are two ways of setting the floorplan dimensions.
+The user can either specify manually die/core area, or
+specify the utilization/aspect ratio.
+
+#### Method 1: Automatical die size calculation
```
Comment (vvbandeira) [inline src/ifp/README.md:18]:
```suggestion
#### Method 1: Automatic die size calculation
```

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**